crunchers/cruncher_active_users.h

```diff
@@ -77,6 +74,13 @@ struct ActiveUsersCruncherImpl {
   using user_list_t = std::list<ActiveUser>;
   using user_map_t = std::unordered_map<std::string, typename user_list_t::iterator>;
 
+  void RemoveUsersOutsideTheTimeWindow() {
+    while (!users_list_.empty() && users_list_.back().us + interval_ <= current_us_) {
```

Comment:
Imagine you have `time_window_size = std::chrono::microseconds(1)`. You publish two events with the timestamps equal to `0` and `1` respectively. How many events do you expect to see in your time window buffer?